appcoins-ads/src/main/java/com/asf/appcoins/sdk/ads/poa/manager/WalletUtils.java

```diff
@@ -106,6 +118,11 @@ public static int getAptoideVersion() {
     return versionCode;
   }
 
+  private static boolean resolveActivityInfoForIntent(Intent intent) {
+    ActivityInfo activityInfo = intent.resolveActivityInfo(context.getPackageManager(), 0);
+    return activityInfo != null;
```

Comment:
You can remove the comparing since it is always negated when called.

for example:
 private static boolean resolveActivityInfoForIntent(Intent intent) {
    ActivityInfo activityInfo = intent.resolveActivityInfo(context.getPackageManager(), 0);
    return activityInfo == null;
  }
and

if (resolveActivityInfoForIntent(intent)) {
      intent = getNotificationIntentForBrowser();
      if (resolveActivityInfoForIntent(intent)) {
        intent = null;
      }
    }